### PR TITLE
nphprediction - DICOM to nii.gz conversion

### PR DESCRIPTION
**Pull request by kthanigaivelan**
The module should now be able to handle any DICOM files that get placed into the Scans directory by first converting them to compressed NIFTI gzip files.
New imports/dependencies: nipype library
New files: convUtils.py
To test: Load .dcm files instead of .nii or .nii.gz (must have all DICOM files corresponding to a CT scan in order to segment properly)

Closes issue #51

### CODE REVIEW

File: source/modules/nphprediction/source/convUtils.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS        ##
+## SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.              ##
+##                                                                           ##
+## The views and conclusions contained in the software and documentation     ##
+## are those of the authors and should not be interpreted as representing    ##
+## official policies, either expressed or implied, of <copyright holder>.    ##
+###############################################################################
+
+def convert_dcm_to_niigz(dir):
+    from nipype.interfaces.dcm2nii import Dcm2niix
```
Comment (amilworks) [inline source/modules/nphprediction/source/convUtils.py:42]: Do we need to add this dependency to the `Dockerfile`? I assume its a `pip install [nipy]`? Unless we already have this library and you have tested this in the Docker container?

File: source/modules/nphprediction/source/convUtils.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    converter = Dcm2niix()
+    converter.inputs.source_dir = dir
+    converter.inputs.out_filename = '%b'
```
Comment (amilworks) [inline source/modules/nphprediction/source/convUtils.py:45]: Enlighten me on what `%b` does and which `Python` version you are using.

**Review by amilworks — CHANGES_REQUESTED** [review]

File: source/modules/nphprediction/source/convUtils.py (hunk 2 above)
Comment (kthanigaivelan) [reply source/modules/nphprediction/source/convUtils.py:45]: So this is for the naming scheme used when the nii file is created - in this case, I’m having it name it the same as the DICOM so we don’t have issues with file names if we want to access/compare for dice and such in the future

File: source/modules/nphprediction/source/convUtils.py (hunk 1 above)
Comment (amilworks) [reply source/modules/nphprediction/source/convUtils.py:42]: Yeah can you add the pip install here. If version does not matter then you can omit it.

https://github.com/UCSB-VRL/bisqueUCSB/blob/015bf9f20dc56ddfe18e12d7c1a4114bf18cc8bc/source/modules/nphprediction/Dockerfile#L58

Comment (kthanigaivelan) [reply source/modules/nphprediction/source/convUtils.py:42]: Added!

**Comment by amilworks** [comment]
Okay this looks a lot better. I will go ahead and build the image for the new NPH Prediction module with these changes. Specifically, this adds functionality for DICOMs as input into the module.

**Pull request merged**